nornir/plugins/inventory/netbox.py

```diff
@@ -36,17 +36,41 @@ def __init__(
         )
         headers = {"Authorization": "Token {}".format(nb_token)}
 
+        def make_request(url: str) -> Any:
+            req = requests.get(url=url, headers=headers, params=filter_parameters)
+            req.raise_for_status()
+
+            if req.ok:
+                return req.json()
+
+        def req_all(url: str) -> Any:
+            req = make_request(url)
+            if isinstance(req, dict) and req.get("results") is not None:
+                ret = req["results"]
+                first_run = True
+                while req["next"]:
+                    next_url = (
+                        "{}{}limit={}&offset={}".format(
```

Comment:
Use f-strings

Reply:
Done.
Since it was originally formatted across multiple lines, I had to use multi-line strings.